http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -221,7 +229,12 @@ class LeoPubsubMessageSubscriber[F[_]](
                                 4 minutes,
                                 4.5 minutes
       )
-      metricsName = s"pubsub/ack/${event.msg.messageType.asString}"
+      messageType = event.msg.messageType.asString
+      metricsName = e match {
+        case Some(e) =>
+          s"pubsub/fail/${messageType}/${e.getClass.toString.split(".").last}"
+        case None => s"pubsub/ack/${messageType}"
+      }
```

Comment:
@lucymcnatt - here is the update to include / capture the error type. my thought here is that this will correlate to the log messaging that happens in `processMessageFailure(...)` and will give us a starting place to get more information. thoughts?

Reply:
I think this is a good starting point, and we can iterate if we need to (if that's too granular/not granular enough)